packages/p2p-v2/src/pages/my-profile/screens/MyProfileCounterparties/MyProfileCounterpartiesEmpty.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Text } from '@deriv-com/ui/dist/components/Text';
+import EmptyCounterparties from '../../../../public/ic-empty-blocked-advertisers.svg';
```

Comment:
small nit, just make sure if its an Icon, to add Icon to the name as well
```suggestion
import EmptyCounterpartiesIcon from '../../../../public/ic-empty-blocked-advertisers.svg';
```